src/main/java/com/sumologic/jenkins/jenkinssumologicplugin/sender/LogSender.java

```diff
@@ -112,6 +127,11 @@ private void createHeaders(final PostMethod post, final String sumoName,
             post.addRequestHeader("Content-Type", contentType);
         }
 
+        if (fields != null && !fields.isEmpty()) {
+            String field_string = fields.keySet().stream().map(key -> key + "=" + fields.get(key)).collect(Collectors.joining(","));
+            post.addRequestHeader("X-Sumo-Fields", field_string);
+        }
+
         post.addRequestHeader("X-Sumo-Client", "sumologic-publisher");
```

Comment:
"sumologic-publisher" sounds generic, Maybe  "sumologic-jenkins-plugin" or something with "jenkins" in there ( as Sumo provides multiple clients)

Reply:
Yes, I can change it to sumologic-jenkins-plugin. Nice catch, I never checked this value.